docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<ul><li>Please capitalize Doppler</li><li>The relative radial velocity will be negative, because the radar is approaching the target.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>